internal/operators/manager.go

```diff
@@ -463,13 +449,24 @@ func EnsureLVMAndCNVDoNotClash(cluster *common.Cluster, openshiftVersion string,
 		}
 	}
 
-	if lvmEnabled && cnvEnabled {
-		return errors.Errorf("Currently, you can not install OpenShift Data Foundation Logical Volume Manager operator at the same time as Virtualization operator.")
+	if !lvmEnabled || !cnvEnabled {
+    return nil
+	}
+
+	// Openshift version greater or Equal to 4.12.0 support cnv and lvms
+  if isGreaterOrEqual, _ := common.BaseVersionGreaterOrEqual(lvm.LvmsMinOpenshiftVersion4_12, openshiftVersion); !isGreaterOrEqual {
+		return errors.Errorf("Currently, you can not install OpenShift Data Foundation Logical Volume Manager operator at the same time as Virtualization operator.") 
 	}
 
-	if lvmEnabled && !common.IsSingleNodeCluster(cluster) {
+  if isGreaterOrEqual, _ := common.BaseVersionGreaterOrEqual(lvm.LvmMinMultiNodeSupportVersion, openshiftVersion); isGreaterOrEqual {
+    return nil
+  }
+
+  // before 4.15 muli node LVM not supported
+	if !common.IsSingleNodeCluster(cluster) {
```

Comment:
the error message "OpenShift Data Foundation Logical Volume Manager operator is only supported for Single Node Openshift" is not correct

Reply:
That was the message before than, What should it be?

Reply:
should be the same message as:
message := fmt.Sprintf("Logical Volume Manager is only supported for highly available openshift with version %s or above", LvmMinMultiNodeSupportVersion) but I don't like the highly available openshift

Reply:
@nirfarkas needs your help with the error message for LVMS on multi node cluster in OCP version < 4.15